pom.xml

```diff
@@ -85,19 +81,42 @@
             <dependency>
                 <groupId>com.google.inject</groupId>
                 <artifactId>guice</artifactId>
-                <version>4.2.2</version>
+                <version>4.2.3</version>
             </dependency>
 
+            <!-- Logging -->
+            <!-- This dependency configuration is to prevent
+                 http://www.slf4j.org/codes.html#StaticLoggerBinder
+                 See https://stackoverflow.com/a/56089477/1187277 -->
             <dependency>
                 <groupId>ch.qos.logback</groupId>
                 <artifactId>logback-classic</artifactId>
-                <version>1.1.8</version>
+                <version>${logback.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-core</artifactId>
+                <version>${logback.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>1.7.30</version>
             </dependency>
 
+            <!-- <dependency> -->
+            <!--     <groupId>org.slf4j</groupId> -->
+            <!--     <artifactId>slf4j-simple</artifactId> -->
+            <!--     <version>1.7.30</version> -->
+            <!-- </dependency> -->
```

Comment:
why do we replace `slf4j-simple` with `slf4j-api`? I guess, it deserves a comment.

Reply:
Oops, my mistake here! I shouldn't have left this commented out. Thanks for the catch. So, iiuc, we are using logback as our logging backend, not slf4j-simple. Including both `simple` and logback backends in dependencies trigger a runtime error fro self4j. I referred to the error and the SO answer explaining the needed configuration on https://github.com/informalsystems/apalache/pull/190/files#diff-600376dffeb79835ede4a0b285078036R88-R90

With this commented out dependency removed, would you still like additional documentation in the comments?